src/main/java/com/fabriccommunity/spookytime/util/Contributors.java

```diff
@@ -1,62 +1,72 @@
 package com.fabriccommunity.spookytime.util;
 
-import com.google.common.collect.ImmutableList;
-
+import com.fabriccommunity.spookytime.SpookyTime;
+import net.fabricmc.loader.api.FabricLoader;
+import net.fabricmc.loader.api.metadata.ContactInformation;
+import net.fabricmc.loader.api.metadata.ModMetadata;
+import net.fabricmc.loader.api.metadata.Person;
 import net.minecraft.server.command.ServerCommandSource;
-import net.minecraft.text.LiteralText;
-import net.minecraft.text.Text;
-import net.minecraft.text.Texts;
-import net.minecraft.text.TranslatableText;
+import net.minecraft.text.*;
 import net.minecraft.util.Formatting;
+import org.apache.commons.lang3.mutable.MutableBoolean;
 
-import java.util.HashMap;
+import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
 
 /**
  * A list of all contributors who helped out in the development process of
  * SpookyTime
  */
 public class Contributors {
-	// Please leave adding names to this when we release so we can have a complete
-	// list
-	
-	private static final Map<String, Formatting> USERS_AND_FORMATTING = new HashMap<>();
-	
-	private static final List<Formatting> DISALLOWED_FORMATTING = ImmutableList.of(Formatting.BOLD,
-			Formatting.OBFUSCATED, Formatting.STRIKETHROUGH, Formatting.UNDERLINE, Formatting.ITALIC);
-	
+	// Please add names to the fabric mod json
+
+	private static final List<Text> USER_INFO = new ArrayList<>();
+
 	static {
 		initContributors();
 	}
-	
+
 	private static void initContributors() {
-		/*
-		 * Here is an example of how to add a user to the list
-		 *
-		 * USERS_AND_FORMATTING.put("Github-Username/MC Username", Formatting.DARK_RED);
-		 *
-		 * Note that Magic, Underline and bold formattings are striped to make sure
-		 * someone doesn't mess up.
-		 */
+		ModMetadata metadata = FabricLoader.getInstance().getModContainer(SpookyTime.MOD_ID).orElseThrow(() -> new IllegalStateException("Spooky time mod broke!")).getMetadata();
```

Comment:
Could we change the string out for a translation? While I have a hard time seeing this error ever being thrown, a more appropriate message could be "Fabric failed to load Spooky Time's mod metadata! Please report this as a bug."

Reply:
Hmm, translation won't be available on the dedicated server 🤔 
And if this does happen it is a loader bug, not really one of ours

Reply:
Ah, didn't know translations couldn't be used in this context.

Reply:
Fixed. Now I'm asking people to shout at loader (if this broke, it is indeed the loader's fault)